example-application/api.js

```diff
@@ -115,7 +122,9 @@ const defineRoutes = (expressApp) => {
       }
     }
     await errorHandler.handleError(error);
-    res.status(500).end();
+
+    const outgoingStatusCode = error?.isAxiosError && error?.code === 'ECONNABORTED' ? 503 : 500;
```

Comment:
I don't think that we should return 503 when our dependency returns 503. This will create a cascading domino failure of a system. Also, this status code is at the service level, not route level, why not accept requests anymore to other routes? 

What elsewhere should we do? This is an interesting distributed system design question... Thinking 🤔

Reply:
Hm.. I didn't think about 503 as a service-level error (as 500 doesn't mean that the whole service is down)
Maybe you're right tho because this is more a convention thing.

Anyway, this domino effect is ok IMO because one of the services in the chain is unavailable.
Also, 503 is nice for this because the server can propagate the `Retry-After` header to its client.